.github/workflows/release.yaml

```diff
@@ -0,0 +1,42 @@
+name: Release Latest Manifests
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches: ['main']
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+  TAG_NAME: latest
+
+jobs:
+  relase-latest-manifests:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Create or Update latest Tag
+        run: |
+          git tag -f ${{ env.TAG_NAME }}
+          git push origin -f ${{ env.TAG_NAME }}
+
```

Comment:
I completely agree.

Btw, we need a way to easily publish the latest version of manifests to use them with `primazactl`. I'd say that we can temporarily use this approach. It mimics the behavior of tag `latest` for container images.
Finally, a tag is mandatory for having a GitHub Release, for this reason we need this step.
As we have an `alpha` release we can modify this workflow to publish manifests when a tag is created.

Do you think we can obtain the same result without tagging?

Reply:
I don't think it's impossible; we could have it push another branch within this repository that hosts the manifests.  For the model you're describing, branches feel more like the abstraction we want here, not tags.

I'd like to at least look at a few other options besides tagging every single commit.  Looking at some other projects, I see that for distributing builds not yet in a release generally fall into a few categories:
- Nightly builds.  At the end of every day (e.g. ~midnight UTC or so), a new release is published based off of whatever is living in `main` at the time.  No guarantees are made of stability; if you get the proverbial nasal demons, well, that's a part of the risk you need to accept.  It's good for testing, but not necessarily for production.
- Frequent releases.  I know of a few projects who make it a point to tag a new patch release fairly frequently and periodically, ranging from nightly (see above) to every two weeks or so.
- Make it easy to consume your git repository directly by your users.  This I see more in libraries, typically in conjunction with the project's build system.  We could do this for primaza, though I'm not sure we'd be getting a lot of value out of it.

(This is by no means a complete list, just ones I know of off-hand)

Are we anticipating on supporting primaza installs from `main` in `primazactl`?  I'm not convinced this is something we have the capability to support out of the gate; while it would be nice to have, I think our efforts might be better spent elsewhere.